flows/index.md

```diff
@@ -0,0 +1,128 @@
+
+## Build the Flow
+Click **Select First Step** and start selecting the components that will bring in data from your selected data sources. Use the drop-down menu to select a particular configuration of the component.
+
+{: .image-popup}
+![Select First Step](/tutorial/automate/automate3.png)
+
+Use the plus icon to add other steps. 
+
+Apart from the extractor(s), you'll want to add selected transformations and writers that will load your transformed data into your selected data destination.
+Continue adding other ready-made components and organize them in a logical flow,  or select those you want to run in parallel using drag-and-drop function. 
+
+- Organize the components into a logical flow using the drag-and-drop functionality.
+- Configure each component by providing the credentials and instructions for what or where to extract or write or for what code to execute in a transformation.
+- Remember to adjust the input mapping of your transformation(s) to use the tables extracted from your selected data sources if necessary. Select the step and click **Edit Configuration**.
+- Save the changes.
+
+Once you’ve built your flow end-to-end, it may look something like this:
+
+{: .image-popup}
+![Add SQL Transformation](/tutorial/automate/automate10.png)
+   
+Click **Run Flow** to start the data pipeline.
+
+## Execute Tasks in Parallel
+You can group multiple tasks within one step, a phase. These tasks then run independently in parallel, speeding up the execution. 
+Steps execute sequentially, while tasks within a single step run in parallel. If you have multiple extractors, you can include them all in a single step, allowing them to run simultaneously. 
+
+The same applies to data writers. Also, transformations independent of the connectors can be grouped within the same step. Note that this does not reduce costs, as each job consume credits independently.
+
+***Warning:** If too many tasks are scheduled in a single phase, you may exceed the available Storage job slots, causing delays in your flow’s execution. Limiting the number of concurrent component jobs 
+to 10 is recommended to avoid reaching Storage capacity limits. You can, of course, configure your flows to execute more jobs in parallel. Keboola will then concurrently execute the jobs to the maximum 
+extent possible based on available resources.*
+
+You can also set up parallelization **among components**, directly in the component's UI for [row-based components](/components/#configuration-rows) like database extractors using the same credentials to run multiple tables concurrently.
+
+{% include tip.html title="Execute Individual Configurations" content="
+Run configurations individually in Keboola Flows for more efficient workflow management. You can fine-tune which rows to run in advanced settings for greater 
+control.
+
+Use the menu next to each configuration row in the UI to execute specific configurations as needed, optimizing time and resources.
```

Comment:
nechápu

Reply:
to mam odsud - https://help.keboola.com/tutorial/onboarding/cheat-sheet/#optimize-with-parallelization

Tak tu vetu vyhodim?

Reply:
stejně to nechápu :D

Reply:
natoz pak ja :-)